application/src/main/kotlin/net/corda/v5/application/marshalling/MarshallingService.kt

```diff
@@ -5,11 +5,9 @@ import net.corda.v5.application.flows.RPCStartableFlow
 import net.corda.v5.base.annotations.DoNotImplement
 
 /**
- * [MarshallingService] marshalls to and from formatting string data.
- *
- * Corda provides a number of marshalling services for converting between string data in different formats. Users should
- * not ask for a [MarshallingService] directly but should instead use one of the specialized services that declare
- * what format to work with (e.g. [JsonMarshallingService] for working with JSON data).
+ * [MarshallingService] is an abstract interface for marshalling to and from formatting string data.
```

Comment:
[MarshallingService] is an abstract interface for marshalling to and from formatted string data.

?

Reply:
Yes, some functions take the abstract MarshallingService as a parameter (see the @see RPCStartableFlow) but nothing implements it directly.

Reply:
Sorry, I should have been clearer. It was the use of "formatting" I wasn't sure of..

Reply:
fixed.